core/modules/date/date.tokens.inc

```diff
@@ -7,19 +7,21 @@
 /**
  * Implements hook_token_info().
  */
-function date_token_info() {
+function date_token_info() {  
```

Comment:
There's some whitespace at the end of the line. Probably slipped in unintetionally.